### PR TITLE
Add IoT images

### PR DESCRIPTION
**Pull request by ctron**

### CODE REVIEW

File: iot-tenant-service/image-template.yaml [hunk 1]
```diff
@@ -0,0 +1,69 @@
+schema_version: 1
+
+version: 1.0
```
Comment (lulf) [inline iot-tenant-service/image-template.yaml]: Should be 1.1

Comment (ctron) [reply iot-tenant-service/image-template.yaml]: Ok, done.

File: modules/common/go/install.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -e
+
+SOURCES_DIR=/tmp/artifacts/
+
+# unpack
+mkdir -p /opt/
+unzip "${SOURCES_DIR}/dist-bin.zip" -d /opt
```
Comment (lulf) [inline modules/common/go/install.sh:9]: Nice, we can probably do this for common.java as well later.

**Review by lulf — APPROVED** [review]
LGTM! Nice work!

**Pull request merged**